.buildkite/schedules/dependency-analysis.yml

```diff
@@ -0,0 +1,23 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+---
+
+# Nodes with values to reuse in the pipeline.
+common_params:
+  # Common plugin settings to use with the `plugins` key.
+  - &common_plugins
+    - automattic/bash-cache#2.11.0
```

Comment:
Do you know where's the source code for this plugin? I thought it's somewhere here https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin but can't find it.

Reply:
So, updating this plugin was done for [Gradle Wrapper Validation](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/34) purposes and it was also mentioned in [2.11.0](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/releases/tag/2.11.0) release.

Not sure what you mean, but let me check if I can find the source code (if any)... 🤔

Reply:
Ok, I found this [PR](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/pull/39), which seems to be renaming that to `a8c-ci-toolkit`. Does that help?

Reply:
@wzieba in case you missed ⬆️ this!

Reply:
So if `bash-cashe` is the same as `a8c-ci-toolkit`, shouldn't we use the latter? Then, we should use the newest version, probably, `3.4.2`. Or is it fine to continue using `bash-cash` name. @AliSoftware, I think you'll know the best 🙂

Reply:
Indeed `bash-cache` was renamed `a8c-ci-toolkit` a while ago. Am actually even surprised that the GitHub redirect for https://github.com/automattic/bash-cache-buildkite-plugin still works even though I think it's been more than a year now? 😄 

We should indeed definitively use its new name `a8c-ci-toolkit` instead here.
From [the migration guide](https://github.com/Automattic/a8c-ci-toolkit-buildkite-plugin/blob/trunk/MIGRATION.md), it seems the only breaking change from `2.x.x` (here `2.11.0`) to the latest `3.4.2` was a change in `nvm_install`… which is not used in this PCAndroid repo, so it should be ok to update it to `a8c-ci-toolkit#3.4.2` directly indeed.

Reply:
Thanks @AliSoftware for explanation!

Reply:
FYI: This is now done: #2487